airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/message/DestinationMessage.kt

```diff
@@ -79,13 +79,17 @@ data class Meta(
     }
 }
 
-data class DestinationRecord(
+sealed interface DestinationRecordPayload : DestinationRecordDomainMessage {
+    val serialized: String
+}
+
+data class DestinationRecordMarshaled(
     override val stream: DestinationStream.Descriptor,
     val data: AirbyteValue,
     val emittedAtMs: Long,
     val meta: Meta?,
-    val serialized: String,
-) : DestinationStreamAffinedMessage {
+    override val serialized: String = "",
```

Comment:
Why is this become optional?

Reply:
Honestly it was just because it was getting set unnecessarily all over the tests, and I'm about to drop it from this class when I break out `...Serialized`

Reply:
I reverted it to required reduce the # of changes here.